pipeline_steps/kibana.groovy

```diff
@@ -15,7 +15,7 @@ def kibana(branch, vm=null){
 
 def kibana_prep(branch){
   dir("kibana-selenium") {
-    git url: "https://github.com/rcbops-qe/kibana-selenium.git", branch: "${branch}"
+    git url: env.KIBANA_SELENIUM_REPO, branch: "${branch}"
```

Comment:
The ability to test implies to me the ability to be able to modify the branch as well as the repo.

It appears the real issue here was that kibana-selenium was out of sync with rpc-openstack, how have processes been improved to prevent this in the future?

The code itself seems fine however I from what I understand of this issue I suspect it will prove to be of limited use. I am happy to see it merged but would first like to understand how the root cause of this issue has been addressed.